drivers/hmis_external_apis/app/jobs/hmis_external_apis/ac_hmis/upload_clients_job.rb

```diff
@@ -4,40 +4,88 @@
 # License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
 ###
 
-# job =  HmisExternalApis::AcHmis::UploadClientsJob.new
+# job = HmisExternalApis::AcHmis::UploadClientsJob.new
 module HmisExternalApis::AcHmis
   class UploadClientsJob < BaseJob
```

Comment:
Can we rename this to `DataWarehouseUploadJob` or something? I think it will be confusing later, since it's actually handling all the warehouse uploads, not just the client